src/dlgTriggerEditor.cpp

```diff
@@ -7078,6 +7084,261 @@ void dlgTriggerEditor::slot_debug_mode()
     mudlet::mpDebugArea->setWindowTitle("Central Debug Console");
 }
 
+void dlgTriggerEditor::slot_next_section()
+{
+    switch (mCurrentView) {
+    case EditorViewType::cmTriggerView:
+        if (QString("edbee::TextEditorComponent").compare(QApplication::focusWidget()->metaObject()->className()) == 0) {
+            treeWidget_triggers->setFocus();
+            return;
+        } else if (treeWidget_triggers->hasFocus()) {
+            mpTriggersMainArea->lineEdit_trigger_name->setFocus();
+            return;
+        } else if (mpTriggersMainArea->hasFocus()) {
+            mTriggerPatternEdit[0]->lineEdit_pattern->setFocus();
+            return;
+        } else {
+            for (auto child : mpTriggersMainArea->scrollArea->findChildren<QWidget *>()) {
+                if (child->hasFocus()){
+                    mpSourceEditorEdbee->setFocus();
+                    return;
+                }
+            }
+            for (auto child : mpTriggersMainArea->findChildren<QWidget *>()) {
+                if (child->hasFocus()){
+                    mTriggerPatternEdit[0]->lineEdit_pattern->setFocus();
+                    return;
+                }
+            }
+        }
```

Comment:
Can you run `clang-format` on this entire function? Not file. Here's an example of how to use in Qt Creator in case you're not familiar (if you are, ignore!):

![Peek 2019-08-16 06-18](https://user-images.githubusercontent.com/110988/63143299-ddbfce00-bfed-11e9-8e8d-5cb33b6da8bb.gif)

Reply:
Done in commit.